packages/typos/package.yaml

```
@@ -0,0 +1,26 @@
+---
+name: typos
+description: Source code spell checker
+homepage: https://github.com/crate-ci/typos
+licenses:
+  - MIT
+  - Apache-2.0
+languages: []
+categories:
+  - Linter
+
+source:
+  id: pkg:github/crate-ci/typos@v1.16.12
+  asset:
+    - target: darwin_x64
+      file: typos-{{ version }}-x86_64-apple-darwin.tar.gz
+      bin: typos
+    - target: linux_x64_gnu
```

This binary seems to be statically linked, so no need to have the GNU constraint:

```suggestion
    - target: linux_x64
```